Makefile

```diff
@@ -152,7 +152,7 @@ godocs:
 test: test_unit
 
 test_cli:
-	@go test -count 1 `go list github.com/cosmos/cosmos-sdk/cmd/gaia/cli_test` -tags=cli_test
+	@go test -p 4 `go list github.com/cosmos/cosmos-sdk/cmd/gaia/cli_test` -tags=cli_test
```

Comment:
Does this really work then?

Reply:
I have had no issues with this. Is it not working for you?

Reply:
It works brilliantly!!